lib/katello/tasks/pulp3_migration.rake

```diff
@@ -47,6 +47,14 @@ namespace :katello do
               exit(-1)
             end
           end
+          repos_missing_errata = ::Katello::Repository.joins("inner join katello_repository_errata on katello_repositories.id = katello_repository_errata.repository_id").
+            joins("inner join katello_root_repositories on katello_repositories.root_id = katello_root_repositories.id").
+            where("katello_repository_errata.erratum_pulp3_href is null").pluck(:repository_id, :name).uniq
+          if repos_missing_errata.present?
+            puts "\nRepositories with the following IDs and names have unmigrated errata:\n"
```

Comment:
I wonder if we should give them a recommendation step, such as 'Resync the repositories'?

Reply:
and maybe re-running the 'content prepare' step with reimport_all=true